.github/workflows/performance.yaml

```diff
@@ -0,0 +1,71 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+name: Performance Tests
+on:
+  push:
+
+env:
+  # uncomment this to enable an ngrok access to the vm on test failure
+  # enabling this you will get an ssh command to access the vm, active for one hour
+  #NGROK_DEBUG: yes
+  # you need to add as secrets an ngrok token and a password to access the terminal 
+  NGROK_TOKEN: ${{ secrets.NGROK_TOKEN }}
```

Comment:
Does this imply those who need debugging should sign up for Ngrok?

Reply:
Yes but the NGROK account is free for one connection.